### ISSUE TITLE
Unsupported extension: DuplicateRecordFields

### ISSUE THREAD

**Issue by ChinaXing** [typelead/eta#512]
src/Parser.hs:3:14: Unsupported extension: DuplicateRecordFields

eta use GHC 7.

but the duplicateRecordFields extension is useful, how to workaround this feature ?

**Comment by rahulmutt** [comment]
Workaround is to use a naming scheme for your record fields :) We will eventually backport this extension for compatibility with GHC 8.

**Comment by ChinaXing** [comment]
ok. can eta easily use GHC 8 'feature, not GHC 7?

GHC 8 is practice, many code developed with GHC 8, and new code will use GHC8.

**Comment by rahulmutt** [comment]
Yes, it's just a matter of back porting the code. Eta and GHC share a common code base so it's a lot easier to do than you would expect. I want to set up a comprehensive test suite before doing to ensure that nothing breaks when doing such backports.
